packages/circuit-ui/components/Input/Input.tsx

```diff
@@ -110,6 +115,42 @@ const containerStyles = () => css`
 
 const InputContainer = styled('div')(containerStyles);
 
+type LabelElProps = Pick<InputProps, 'noMargin'>;
+
+const labelCustomStyles = ({ theme }: StyleProps) => css`
+  label: input__label;
+
+  label &:not(label),
+  label + &:not(label) {
+    margin-top: ${theme.spacings.bit};
+  }
+`;
```

Comment:
Oops, this was removed in the context of requiring the label prop and should not be reintroduced